front/app/containers/ProjectsShowPage/shared/header/ProjectInfoSideBar/index.tsx

```diff
@@ -256,24 +250,17 @@ const ProjectInfoSideBar = memo<Props>(({ projectId, className }) => {
                     {...messages.xParticipants}
                     values={{ participantsCount: projectParticipantsCount }}
                   />
-                  {
-                    <Box mb="4px" ml="4px">
-                      {hasSurveyWithAnyonePermissions(
-                        projectPermissions,
-                        phasesPermissions
-                      ) &&
-                        isAdminUser && (
-                          <IconTooltip
-                            placement="top-start"
-                            maxTooltipWidth={200}
-                            iconColor={colors.coolGrey300}
-                            content={formatMessage(
-                              messages.participantsTooltip
-                            )}
-                          />
-                        )}
-                    </Box>
-                  }
+                  {hasNativeSurvey(project.data, phases?.data) &&
```

Comment:
This tooltip tells you that the 'participants count' includes anonymous survey submissions (i.e. ones that were made in phases that have surveys that have 'everyone' as access right).

Before, this would only be shown if we know that at least one survey phase in this project has access rights set to 'everyone'.

Now, this is shown if any phase has a survey, and I modified the copy a bit to explain that 'anonymous survey responses' happen if you set access rights to 'everyone'.

This saves us tons of requests for a feature that only admins could see anyway